lib/services/eventgrid/package.json

```diff
@@ -14,12 +14,12 @@
   "license": "MIT",
   "main": "./lib/eventGridClient.js",
   "types": "./lib/eventGridClient.d.ts",
-  "homepage": "https://github.com/azure/azure-sdk-for-node/tree/master/lib/services/eventgrid",
+  "homepage": "https://github.com/azure/azure-sdk-for-node/lib/services/eventgrid",
```

Comment:
Looks like a bug in the generator. @kalyanaj - Can you manually revert the change on this line for now.

Reply:
Thanks @amarzavery for the review, I have manually reverted this change on this line and have pushed the changes.